Resources/Prototypes/Entities/Constructible/Furniture/beds.yml

```diff
@@ -41,3 +41,76 @@
           MaterialCloth1:
             min: 1
             max: 1
+
+
+
+- type: entity
+  id: RollerBed
+  parent: BaseConstructibleDynamic
+  name: rollerbed
+  description: Used to carry patients around without damaging them.
+  components:
+    - type: Item
+      size: 5
+    - type: Sprite
+      sprite: Constructible/Misc/furniture.rsi
+      state: rollerbed
+      netsync: false
+    - type: MovedByPressure
+    - type: DamageOnHighSpeedImpact
+      soundHit: /Audio/Effects/bang.ogg
+    - type: InteractionOutline
+    - type: Physics
+      fixtures:
+        - shape:
+            !type:PhysShapeAabb
+            bounds: "-0.45, -0.45, 0.05, 0.45"
```

Comment:
This is suss because when rotating shuttles are common mobs are always going to be upright from your perspective. By making the AABB not equilateral means the roller bed cant always be upright (or else the AABB wont match the sprite) and nor can the player mob be always upright when buckled.

Reply:
Rollerbeds don't rotate in SS13, I figured it would be the same for now until someone sprites different directions

Reply:
The problem is that the rollerbed is probably always going to be upright regardless of perspective which means it has to be a box and not a rectangle.